build_for_ultimaker.sh

```diff
@@ -39,7 +39,7 @@ setup_emulation_support()
         fi
     done
 
-    if [ ! -x "${ARM_EMU_BIN}" ]; then
+    if [ ! -x "${ARM_EMU_BIN}" ] && [ "${ARM_EMU_BIN#/}" != "${ARM_EMU_BIN}" ]; then
```

Comment:
I am confused.. If you cannot execute the arm binary, and the arm binary is not equal to the arm binary without preceding slash.. then error. 
In this script the ARM_EMU_BIN should always have the full path including the starting '/'

Reply:
exactly, the ARM_EMU_BIN should always be an absolute path, thus always start with a /. This should be the default anyway.

The alternative is to add the / to the ROOTFS_DIR. which after sleeping over it, I think is better, albeit a bigger patch. Then we can support both absolute and relative paths.